treat Serverless Tier & Dedicated Tier traffic filter rule different (#11189)
pingcap/docs: tidb-cloud/integrate-tidbcloud-with-vercel.md

### All IP addresses allowed for traffic filter in TiDB Cloud

> For Dedicated Tier clusters, make sure that the traffic filter of the cluster allows all IP addresses (set to `0.0.0.0/0`) for connection, this is because Vercel deployments use [dynamic IP  addresses](https://vercel.com/guides/how-to-allowlist-deployment-ip-address). If you use the TiDB Cloud Vercel integration, TiDB Cloud automatically adds a `0.0.0.0/0` traffic filter to your cluster in the integration workflow if there is none.

Reviewer: ```suggestion
For Dedicated Tier clusters, make sure that the traffic filter of the cluster allows all IP addresses (set to `0.0.0.0/0`) for connection, this is because Vercel deployments use [dynamic IP addresses](https://vercel.com/guides/how-to-allowlist-deployment-ip-address). If you use the TiDB Cloud Vercel integration, TiDB Cloud automatically adds a `0.0.0.0/0` traffic filter to your cluster in the integration workflow if there is none.
```

> Serverless Tier clusters allow all IP addresses to connect by default. So users do not need to care about it. 

Author: ```suggestion
Serverless Tier clusters allow all IP addresses for connection by default, so you do not need to configure any traffic filter.
```

> Serverless Tier clusters allows all IP addresses to connect by default. So users do not need to care about it. 

Reviewer 2: ```suggestion
Serverless Tier clusters allow all IP addresses to connect by default. So users do not need to care about it. 
```

> Serverless Tier clusters allows all IP addresses to connect by default. So users do not need to care about it. 

Reviewer: ```suggestion
Serverless Tier clusters allows all IP addresses for connection by default, so you do not need to configure any traffic filter.
```
